grpc build is broken

https://oss-fuzz-build-logs.storage.googleapis.com/index.html

```
Step #4: + clang++ -O1 -fno-omit-frame-pointer -gline-tables-only -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address -fsanitize-address-use-after-scope -fsanitize=fuzzer-no-link -stdlib=libc++ -Iinclude -I. -stdlib=libc++ test/core/client_channel/uri_fuzzer_test.cc -o /workspace/out/address/uri_fuzzer_test -lFuzzingEngine bazel-bin/libalts_frame_protector.a bazel-bin/libalts_proto.a bazel-bin/libalts_util.a bazel-bin/libcensus.a bazel-bin/libgpr_base.a bazel-bin/libgrpc++.a bazel-bin/libgrpc++_base.a bazel-bin/libgrpc++_base_unsecure.a bazel-bin/libgrpc++_codegen_base_src.a bazel-bin/libgrpc++_core_stats.a bazel-bin/libgrpc++_error_details.a bazel-bin/libgrpc++_unsecure.a bazel-bin/libgrpc.a bazel-bin/libgrpc_base.a bazel-bin/libgrpc_base_c.a bazel-bin/libgrpc_client_channel.a bazel-bin/libgrpc_cronet.a bazel-bin/libgrpc_csharp_ext.a bazel-bin/libgrpc_deadline_filter.a bazel-bin/libgrpc_http_filters.a bazel-bin/libgrpc_lb_policy_grpclb.a bazel-bin/libgrpc_lb_policy_grpclb_secure.a bazel-bin/libgrpc_lb_policy_pick_first.a bazel-bin/libgrpc_lb_policy_round_robin.a bazel-bin/libgrpc_lb_subchannel_list.a bazel-bin/libgrpc_max_age_filter.a bazel-bin/libgrpc_message_size_filter.a bazel-bin/libgrpc_nanopb.a bazel-bin/libgrpc_plugin_support.a bazel-bin/libgrpc_resolver_dns_ares.a bazel-bin/libgrpc_resolver_dns_native.a bazel-bin/libgrpc_resolver_fake.a bazel-bin/libgrpc_resolver_sockaddr.a bazel-bin/libgrpc_secure.a bazel-bin/libgrpc_server_backward_compatibility.a bazel-bin/libgrpc_server_load_reporting.a bazel-bin/libgrpc_trace.a bazel-bin/libgrpc_transport_chttp2.a bazel-bin/libgrpc_transport_chttp2_alpn.a bazel-bin/libgrpc_transport_chttp2_client_connector.a bazel-bin/libgrpc_transport_chttp2_client_insecure.a bazel-bin/libgrpc_transport_chttp2_client_secure.a bazel-bin/libgrpc_transport_chttp2_server.a bazel-bin/libgrpc_transport_chttp2_server_insecure.a bazel-bin/libgrpc_transport_chttp2_server_secure.a bazel-bin/libgrpc_transport_cronet_client_secure.a bazel-bin/libgrpc_transport_inproc.a bazel-bin/libgrpc_unsecure.a bazel-bin/libgrpc_workaround_cronet_compression_filter.a bazel-bin/libtsi.a bazel-bin/libtsi_interface.a bazel-bin/test/core/util/libfuzzer_corpus_test.a bazel-bin/test/core/util/libgpr_test_util.a bazel-bin/test/core/util/libgrpc_debugger_macros.a bazel-bin/test/core/util/libgrpc_test_util_base.a bazel-bin/test/core/end2end/libcq_verifier.a bazel-bin/test/core/end2end/libend2end_tests.a bazel-bin/test/core/end2end/libhttp_proxy.a bazel-bin/test/core/end2end/libproxy.a bazel-bin/test/core/end2end/libssl_test_data.a bazel-bin/external/boringssl/libssl.a bazel-bin/external/boringssl/libcrypto.a bazel-bin/external/com_github_cares_cares/libares.a bazel-bin/external/com_github_madler_zlib/libz.a bazel-bin/third_party/nanopb/libnanopb.a bazel-bin/libalts_frame_protector.a bazel-bin/libalts_proto.a bazel-bin/libalts_util.a bazel-bin/libcensus.a bazel-bin/libgpr_base.a bazel-bin/libgrpc++.a bazel-bin/libgrpc++_base.a bazel-bin/libgrpc++_base_unsecure.a bazel-bin/libgrpc++_codegen_base_src.a bazel-bin/libgrpc++_core_stats.a bazel-bin/libgrpc++_error_details.a bazel-bin/libgrpc++_unsecure.a bazel-bin/libgrpc.a bazel-bin/libgrpc_base.a bazel-bin/libgrpc_base_c.a bazel-bin/libgrpc_client_channel.a bazel-bin/libgrpc_cronet.a bazel-bin/libgrpc_csharp_ext.a bazel-bin/libgrpc_deadline_filter.a bazel-bin/libgrpc_http_filters.a bazel-bin/libgrpc_lb_policy_grpclb.a bazel-bin/libgrpc_lb_policy_grpclb_secure.a bazel-bin/libgrpc_lb_policy_pick_first.a bazel-bin/libgrpc_lb_policy_round_robin.a bazel-bin/libgrpc_lb_subchannel_list.a bazel-bin/libgrpc_max_age_filter.a bazel-bin/libgrpc_message_size_filter.a bazel-bin/libgrpc_nanopb.a bazel-bin/libgrpc_plugin_support.a bazel-bin/libgrpc_resolver_dns_ares.a bazel-bin/libgrpc_resolver_dns_native.a bazel-bin/libgrpc_resolver_fake.a bazel-bin/libgrpc_resolver_sockaddr.a bazel-bin/libgrpc_secure.a bazel-bin/libgrpc_server_backward_compatibility.a bazel-bin/libgrpc_server_load_reporting.a bazel-bin/libgrpc_trace.a bazel-bin/libgrpc_transport_chttp2.a bazel-bin/libgrpc_transport_chttp2_alpn.a bazel-bin/libgrpc_transport_chttp2_client_connector.a bazel-bin/libgrpc_transport_chttp2_client_insecure.a bazel-bin/libgrpc_transport_chttp2_client_secure.a bazel-bin/libgrpc_transport_chttp2_server.a bazel-bin/libgrpc_transport_chttp2_server_insecure.a bazel-bin/libgrpc_transport_chttp2_server_secure.a bazel-bin/libgrpc_transport_cronet_client_secure.a bazel-bin/libgrpc_transport_inproc.a bazel-bin/libgrpc_unsecure.a bazel-bin/libgrpc_workaround_cronet_compression_filter.a bazel-bin/libtsi.a bazel-bin/libtsi_interface.a
Step #4: bazel-bin/libgrpc_resolver_dns_ares.a(dns_resolver_ares.pic.o): In function `grpc_resolver_dns_ares_init()':
Step #4: /proc/self/cwd/src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc:463: undefined reference to `address_sorting_init'
Step #4: bazel-bin/libgrpc_resolver_dns_ares.a(dns_resolver_ares.pic.o): In function `grpc_resolver_dns_ares_shutdown()':
Step #4: /proc/self/cwd/src/core/ext/filters/client_channel/resolver/dns/c_ares/dns_resolver_ares.cc:481: undefined reference to `address_sorting_shutdown'
Step #4: bazel-bin/libgrpc_resolver_dns_ares.a(grpc_ares_wrapper.pic.o): In function `grpc_cares_wrapper_address_sorting_sort(grpc_lb_addresses*)':
Step #4: /proc/self/cwd/src/core/ext/filters/client_channel/resolver/dns/c_ares/grpc_ares_wrapper.cc:132: undefined reference to `address_sorting_rfc_6724_sort'
Step #4: clang-7.0: error: linker command failed with exit code 1 (use -v to see invocation)
Finished Step #4
ERROR
ERROR: build step 4 "gcr.io/oss-fuzz/grpc" failed: exit status 1
```

--- a/projects/grpc/build.sh
+++ b/projects/grpc/build.sh
@@ -44,6 +44,7 @@ bazel-bin/external/boringssl/libssl.a \
 bazel-bin/external/boringssl/libcrypto.a \
 bazel-bin/external/com_github_cares_cares/*.a \
 bazel-bin/external/com_github_madler_zlib/*.a \
+bazel-bin/third_party/address_sorting/*.a \
 bazel-bin/third_party/nanopb/*.a \
 bazel-bin/*.a \
 "
@@ -55,7 +56,8 @@ EXTERA_BAZEL_FLAGS="--strip=never  $(for f in $CXXFLAGS; do if [ $f != "-stdlib=
 bazel build --dynamic_mode=off --spawn_strategy=standalone --genrule_strategy=standalone \
   $EXTERA_BAZEL_FLAGS \
   $NO_VPTR \
-  :all test/core/util/... test/core/end2end/... third_party/nanopb/... @boringssl//:all @com_github_madler_zlib//:all @com_github_cares_cares//:all
+  :all test/core/util/... test/core/end2end/... third_party/address_sorting/... \
+  third_party/nanopb/... @boringssl//:all @com_github_madler_zlib//:all @com_github_cares_cares//:all
 
 CFLAGS="${CFLAGS} -Iinclude -I."
 CXXFLAGS="${CXXFLAGS} -Iinclude -I. -stdlib=libc++"

